apple/RNGestureHandler.mm

```diff
@@ -495,6 +506,50 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
   return NO;
 }
 
+- (BOOL)areScrollViewRecognizersCompatible:(UIGestureRecognizer *)gestureRecognizer
+                           otherRecognizer:(UIGestureRecognizer *)otherGestureRecognizer
+{
+  if ([self isUIScrollViewPanGestureRecognizer:otherGestureRecognizer] &&
+      [gestureRecognizer isKindOfClass:[RNDummyGestureRecognizer class]]) {
+    UIScrollView *scrollView = [self retrieveScrollView:gestureRecognizer.view];
+    if (scrollView && scrollView == otherGestureRecognizer.view) {
+      return YES;
+    }
+  }
+
+  return NO;
+}
+
+// is UIPanGestureRecognizer and has scrollView property
+- (BOOL)isUIScrollViewPanGestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
```

Comment:
This may also break on macOS.

I guess since there is no touch interaction with scroll views on macOS this entire method can simply accept any `NSGestureRecognizer` and return `NO` on mac (cc. @m-bert)

Reply:
I agree. Right now macOS is still in early stage of development, so we don't have to worry about this logic at the moment 😅